lib/ppx/ppx_js.cppo.ml

```diff
@@ -0,0 +1,412 @@
+open Ast_mapper
+open Ast_helper
+open Asttypes
+open Parsetree
+
+open Ast_convenience
+
+(** Check if an expression is an identifier and returns it.
+    Raise a Location.error if it's not.
+*)
+let exp_to_string = function
+  | {pexp_desc= Pexp_ident {txt = Longident.Lident s}} -> s
```

Comment:
"oups" indeed. Maybe just for attributes ? it's probably not really worth it then.